android/app/src/main/java/com/microsoft/codepush/common/datacontracts/CodePushDeploymentStatusReport.java

```diff
@@ -34,10 +34,8 @@
     @SerializedName("status")
     private CodePushDeploymentStatus status;
 
-    /**
-     * Non-serializable field, temporarily stores information about installed/failed package.
-     */
-    private transient CodePushLocalPackage localPackage;
+    @SerializedName("package")
+    private transient CodePushPackage aPackage;
```

Comment:
Is there a particular reason we use `CodePushPackage` here instead of `CodePushLocalPackage`?

Reply:
@annakocheshkova yes, originally client Acquisition SDK operates with `Package` interface which is equal to `CodePushPackage`